### PR TITLE
Update readme rv

### PR DESCRIPTION
**Pull request by annacuomo**
Ready for review!

PR updating the README to all the new changes, particularly around the SAIGE-QTL RV pipeline.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,74 +77,158 @@ Outputs:
...
-* ```variance_ratio_path```: path to variance ratio txt file generated in step 1
-* ```min_maf```: minimum minor allele frequency (MAF) (default: 0)
-* ```min_mac```: minimum minor allele count (MAC) (default: 5)
-* ```loco_bool```: boolean specifying whether leave-one-chromosome-out should be used (default: ```FALSE```)
-* ```n_markers```: internal parameter to batchify variants tested (default: 10000),
-* ```spa_cutoff```: internal parameter to do with the saddlepoint approximation, does not make much of a difference for us (default: 10000),
+In script (`saige_assoc.py`):
+
+* `vcf_file`: path to VCF file containing genetic variants to test.
+* `vcf_file_index`: corresponding .csi index file (not .tbi). check, just prefix?
```
Comment (Alexander-Stuckey) [inline README.md:175]: Have you checked if it can be just prefix? (ditto for the other places that this is mentioned)

File: README.md [hunk 2]
```diff
@@ -54,74 +77,158 @@ Outputs:
...
+* `saige_output_file`: path to output file (check name).
```
Comment (Alexander-Stuckey) [inline README.md:176]: Have you checked the name? Or is this a warning to other users that they need to check the name? (Ditto for other mentions of check name)

File: README.md [hunk 3]
```diff
@@ -54,74 +77,158 @@ Outputs:
...
+* `chrom`: chromosome to test.
+* `cis_window_file`: path to file specifying cis window / region to test (generated in part 2 of the pipeline, get anndata script).
+* `gmmat_model_path`: path to estimated null model (.rda) generated in step 1.
+* `variance_ratio_path`: path to variance ratio txt file generated in step 1.
+
+In config (under `[saige.sv_test]` in `saige_assoc_test.toml`):
+
+* `vcfField`: DS for dosages, GT for genotypes (default: `GT`).
+* `minMAF`: minimum minor allele frequency (MAF) (default: 0).
+* `minMAC`: minimum minor allele count (MAC) (default: 5).
```
Comment (Alexander-Stuckey) [inline README.md:186]: What happens when both values are set? Does it take the largest?

**Review by Alexander-Stuckey — CHANGES_REQUESTED** [review]
Left a few comments that would be good to get sorted out first, to clarify the readme

File: README.md (hunk 1 above)
Comment (annacuomo) [reply README.md:175]: thanks, I had been meaning to check this and then forgot 🤦‍♀️

File: README.md (hunk 3 above)
Comment (annacuomo) [reply README.md:186]: good question, I assume so? I'll check.

File: README.md (hunk 1 above)
Comment (annacuomo) [reply README.md]: all done now

File: README.md (hunk 2 above)
Comment (annacuomo) [reply README.md]: updated but not very happy with the name, will fix that separately

File: README.md (hunk 3 above)
Comment (annacuomo) [reply README.md]: all done

**Review by Alexander-Stuckey — APPROVED** [review]
Happy with the changes, looks good to me

**Pull request merged**